scripts/data-lint.js

```diff
@@ -0,0 +1,39 @@
+const inspect = require("util").inspect;
+
+const lib = require("./data-lint-lib");
+
+let errors = [];
+
+main();
+
+async function main() {
+	const { communitySchema, demosSchema, pluginsSchema, sitesSchema, startersSchema } = lib;
```

Comment:
We could also just destructure this on L3 [along w/ `schemaLint`], if you prefer.